worlds/hk/docs/setup_en.md

```diff
@@ -3,32 +3,34 @@
 ## Required Software
 * Download and unzip the Lumafly Mod Manager from the [Lumafly website](https://themulhima.github.io/Lumafly/).
 * A legal copy of Hollow Knight.
+   * Steam, Gog, and Xbox Game Pass versions of the game are supported.
+   * Windows, Mac, and Linux (including Steam deck) are supported.
 
 ## Installing the Archipelago Mod using Lumafly
 1. Launch Lumafly and ensure it locates your Hollow Knight installation directory.
 2. Click the "Install" button near the "Archipelago" mod entry.
    * If desired, also install "Archipelago Map Mod" to use as an in-game tracker.
 3. Launch the game, you're all set!
 
-### What to do if Lumafly fails to find your XBox Game Pass installation directory
-1. Enter the XBox app and move your mouse over "Hollow Knight" on the left sidebar. 
-2. Click the three points then click "Manage".
-3. Go to the "Files" tab and select "Browse...". 
-4. Click "Hollow Knight", then "Content", then click the path bar and copy it.
-5. Run Lumafly as an administrator and, when it asks you for the path, paste what you copied in step 4.
-
-#### Alternative Method:
-1. Click on your profile then "Settings". 
-2. Go to the "General" tab and select "CHANGE FOLDER".
-3. Look for a folder where you want to install the game (preferably inside a folder on your desktop) and copy the path.
-4. Run Lumafly as an administrator and, when it asks you for the path, paste what you copied in step 3.
-
-Note: The path folder needs to have the "Hollow Knight_Data" folder inside. 
+### What to do if Lumafly fails to find your installation directory
+1. Find the directory manually.
+   * Xbox Game Pass:
+      1. Enter the XBox app and move your mouse over "Hollow Knight" on the left sidebar. 
+      2. Click the three points then click "Manage".
+      3. Go to the "Files" tab and select "Browse...". 
+      4. Click "Hollow Knight", then "Content", then click the path bar and copy it.
+   * Steam:
+      1. You likely put your Steam library in a non-standard place. If this is the case, you probably know where 
+         it is. Find your steam library and then find the Hollow Knight folder and copy the path.
+         * Windows - C:\Program Files (x86)\Steam\steamapps\common\Hollow Knight
```

Comment:
I think it would make sense to list the example paths with inline code, at least so that the `-` doesn't seem like it could be part of it
```suggestion
         * Windows - `C:\Program Files (x86)\Steam\steamapps\common\Hollow Knight`
```